deploy/assisted-installer-controller/assisted-installer-controller-role.yaml

```diff
@@ -12,10 +12,13 @@ rules:
       - pods
       - configmaps
       - pods/log
+      - services
     verbs:
       - get
       - list
       - watch
+      - delete
```

Comment:
you should add it for services only and not for all other resources

Reply:
done